### PR TITLE
[EXEC-3304]: execution-doc-updates

### PR DESCRIPTION
**Pull request by brentmmarks**
# Description
A brief description of the changes.

Part 1 in updating docs. More changes need to come, but need somewhere to start.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/v4.3/air-gapped-installation/phase-1-prerequisites.adoc [hunk 1]
```diff
@@ -86,19 +86,12 @@ cciserver.azurecr.io/nomad:1.1.2-74-7552671
...
-cciserver.azurecr.io/output-processing:0.1.66937-9609a831
 cciserver.azurecr.io/permissions-service-migrator:0.1.13413-18c6091
 cciserver.azurecr.io/permissions-service:0.1.13412-18c6091
-cciserver.azurecr.io/picard-dispatcher:0.1.65675-630fe5d4
 cciserver.azurecr.io/policy-service:0.1.5903-bf94009
-cciserver.azurecr.io/schedulerer:0.1.65674-630fe5d4
 cciserver.azurecr.io/server-license:0.1.113-be11908
 cciserver.azurecr.io/test-results:0.1.3402-e561bf7
 cciserver.azurecr.io/vault-cci:0.4.196-1af3417
-cciserver.azurecr.io/vm-gc:0.1.116274-09867360
```
Comment (brentmmarks) [inline jekyll/_cci2/server/v4.3/air-gapped-installation/phase-1-prerequisites.adoc:98]: New ones need to be updated - but going to leave this for when list of other pieces is brought over

File: jekyll/_cci2/server/v4.3/installation/installation-reference.adoc [hunk 2]
```diff
@@ -1271,78 +1285,6 @@ a|Session Cookie Key. NOTE: Must be exactly 16 bytes.
...
-|list
-|`[{"count":0,"cron":"","docker-engine":true,"image":"docker-default","type":"l1.medium"},{"count":0,"cron":"","docker-engine":false,"image":"default","type":"l1.medium"},{"count":0,"cron":"","docker-engine":false,"image":"docker","type":"l1.large"},{"count":0,"cron":"","docker-engine":false,"image":"windows-default","type":"windows.medium"}]`
-|Configuration options for, and numbers of, prescaled instances.
-
-|`vm_scaler.replicas`
-|int
-|`1`
-|Number of replicas to deploy for the vm-scaler deployment.
-
-|`vm_service.dlc_lifespan_days`
```
Comment (brentmmarks) [inline jekyll/_cci2/server/v4.3/installation/installation-reference.adoc:1289]: going to update values in readme and have them moved over later with other changes

File: jekyll/_cci2/server/v4.3/air-gapped-installation/phase-1-prerequisites.adoc (hunk 1 above)
Comment (ryanwohara) [reply jekyll/_cci2/server/v4.3/air-gapped-installation/phase-1-prerequisites.adoc:98]: We can use this to generate the list of images when we're ready https://github.com/circleci/server/tree/main/scripts/generate-images-script

File: jekyll/_cci2/server/v4.3/air-gapped-installation/example-values.adoc [hunk 3]
```diff
@@ -121,4 +121,13 @@ rabbitmq:
     password: "<<rabbitmq-password>>"
     erlangCookie: "<<rabbitmq-erlang-cookie>>"
 
+docker_provisioner:
+  providers:
+    ec2:
+      enabled: true
```
Comment (DominicLavery) [inline jekyll/_cci2/server/v4.3/air-gapped-installation/example-values.adoc:127]: I don't think this wants to be true in air gapped

File: jekyll/_cci2/server/v4.3/installation/upgrade-server.adoc [hunk 4]
```diff
@@ -18,6 +18,31 @@ A successful deployment will update the web app. Unless noted in the release not
 
 NOTE: We recommend that you do not skip releases when upgrading.
 
+NOTE: Related to new execution systems
+* Docker Layer Caching (DLC)
+** old docker layer caching (dlc) volumes will not be carried over and need to be wiped manually
+** dlc usage delete threshold is currently not configurable at this time
+** dlc prune days is currently not configurable at this time
+** dlc volume size is currently not configurable at this time
+** dlc wont get cleaned up on s3 (projects that stop using dlc wont delete final DLC cache without manual intervention)
```
Comment (DominicLavery) [inline jekyll/_cci2/server/v4.3/installation/upgrade-server.adoc:27]:
```suggestion
** projects that stop using dlc wont delete final DLC cache without manual intervention
```
I'm not sure the first part is clear, it kind of suggests that the s3 bucket will just infinitely grow. The bit in brackets explains the situation great I think

File: jekyll/_cci2/server/v4.3/operator/operator-overview.adoc [hunk 5]
```diff
@@ -14,9 +14,9 @@ contentTags:
 
 This guide contains information for CircleCI server operators, or those responsible for ensuring CircleCI server is running properly through maintenance and monitoring. Before reading this operator guide, ensure you have read the xref:../overview/circleci-server-overview#[CircleCI server v4.3 overview].
 
-CircleCI server schedules CI/CD jobs using the link:https://www.nomadproject.io/[Nomad] scheduler. The Nomad Server (control plane) runs inside the Kubernetes cluster, while Nomad clients are provisioned outside the cluster. The Nomad clients need access to the Nomad control plane, output processor, and VM service.
+CircleCI server schedules CI/CD jobs using the link:https://www.nomadproject.io/[Nomad] scheduler. The Nomad Server (control plane) runs inside the Kubernetes cluster, while Nomad clients are provisioned outside the cluster. The Nomad clients need access to the Nomad control plane.
```
Comment (DominicLavery) [inline jekyll/_cci2/server/v4.3/operator/operator-overview.adoc:17]:
```suggestion
CircleCI server schedules CI/CD jobs using the link:https://www.nomadproject.io/[Nomad] scheduler. The Nomad Server (control plane) runs inside the Kubernetes cluster, while Nomad clients are provisioned outside the cluster. The Nomad clients need access to the Nomad control plane and Kong.
```

File: jekyll/_cci2/server/v4.3/installation/upgrade-server.adoc [hunk 6]
```diff
@@ -18,6 +18,31 @@ A successful deployment will update the web app. Unless noted in the release not
...
+
+* Machine Jobs
+** machine jobs will now run through machine provisioner
+** When migrating vm-gc will need to cleanup all machines left running before it is removed. All other vm-service related
+pods can be removed during migration. If this is not followed vm-service machines will not be cleaned up without manual intervention.
+This can alternatively be done by just manually cleaning up remaining vms before starting machine-provisioner.
+** Machine and Remote docker jobs no longer flow through nomad and run completely in a single machine. Nomad is NOT required to run machine jobs.
+** Scaling overrides for pre-scaling are currently not exposed at this time.
+** If deploying in specific zone need to specify subnetwork for gcp
+** IAM users have changed names to end with machine-provisoner instead of vm-service so some users might need to be altered.
```
Comment (brentmmarks) [inline jekyll/_cci2/server/v4.3/installation/upgrade-server.adoc:38]: @DomParfitt this is where have let customers know about this change for IAM

**Review by DominicLavery — APPROVED** [review]
Looks good, once rebased

File: jekyll/_cci2/server/v4.3/air-gapped-installation/phase-1-prerequisites.adoc (hunk 1 above)
Comment (brentmmarks) [reply jekyll/_cci2/server/v4.3/air-gapped-installation/phase-1-prerequisites.adoc:98]: going to leave as other changes are going to happen with other teams and dont want to mess something up

**Pull request merged**